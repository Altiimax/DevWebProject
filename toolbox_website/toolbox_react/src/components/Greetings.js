import React from 'react';
/**
 * This component is used to render a personalised greeting message.
 * It rendre the greetings message in h1.
 * @param {*} props 
 */

const Greet = (props) =>{
    return(
    //ici le /n ne fonctionne pas!
    <div>
        <h3 className='Text'>Welcome to the application {props.name}! 
        This application is on construction!
        {"\n"} What would you do?</h3>
    </div>
    );
}
export default Greet;