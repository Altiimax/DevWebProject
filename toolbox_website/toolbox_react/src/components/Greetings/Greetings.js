import React from "react";
import "./Greetings.css";
/**
 * This component is used to render a personalised greeting message.
 * It rendre the greetings message in a h1 element.
 * @param {*} props
 */

const Greet = (props) => {
  return (
    <div>
      <h1 className="Text">Welcome to your profile page {props.name}!</h1>
    </div>
  );
};
export default Greet;
