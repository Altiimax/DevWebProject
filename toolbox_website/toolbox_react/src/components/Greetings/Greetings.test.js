import React from "react";
import ReactDOM from "react-dom";
import Greetings from "./Greetings.js";
it("render without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Greetings />, div);
});
