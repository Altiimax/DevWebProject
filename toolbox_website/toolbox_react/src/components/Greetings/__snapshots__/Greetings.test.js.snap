// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a text 1`] = `
<div>
  <h1
    className="Text"
  >
    Welcome to your profile page 
    !
  </h1>
</div>
`;
